lib/server/controllers/qcFlag.controller.js

```diff
@@ -273,6 +273,28 @@ const getGaqQcFlagsHandler = async (request, response) => {
     }
 };
 
+// eslint-disable-next-line valid-jsdoc
+/**
+ * Get GAQ summary for given data pass
+ */
+const getGaqSummaryHandler = async (request, response) => {
+    const validatedDTO = await dtoValidator(
+        DtoFactory.queryOnly(Joi.object({ dataPassId: Joi.number().required() })),
+        request,
+        response,
+    );
+    if (validatedDTO) {
+        try {
+            const { dataPassId } = validatedDTO.query;
+
+            const data = await qcFlagService.getGaqSummary(dataPassId);
```

Comment:
Would it perhaps be better to check that provided `dataPassId` exists ?

Reply:
applied